gnome-image-installer/gnome-image-installer.c

```diff
@@ -240,6 +240,36 @@ mount_and_read_keys (void)
     }
 }
 
+static gchar *
+get_image_uuid (void)
+{
+  gchar *uuid = NULL;
+  GError *error = NULL;
+  g_autofree gchar *cmdline = NULL;
+  GRegex *reg;
+  GMatchInfo *info;
+
+  if (!g_file_get_contents ("/proc/cmdline", &cmdline, NULL, &error))
```

Comment:
It's a bit sad to be reading and parsing `/proc/cmdline` twice. Maybe merge this function with `check_for_live_boot` below and as a bonus use the value of `EI_FORCE_LIVE_BOOT` as the UUID if set?